.config/dotnet-tools.json

```diff
@@ -7,6 +7,12 @@
       "commands": [
         "jb"
       ]
+    },
+    "docfx": {
```

Comment:
WE can probably get rid of DocFx altogether (it was just part of the template for this repo) but I appreciate you doing this

Reply:
Will probably make CI faster. I figured it could be executed from some over place where the documentation is built, so I tried to keep it comparability reasons